apps/git/git.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+SCRIPT_PATH=$(realpath ${BASH_SOURCE})
+sudo rm -f $SCRIPT_PATH
+
+if [ ! -f /usr/bin/git ]; then
+	sudo apt-get update
+	sudo DEBIAN_FRONTEND=noninteractive apt-get -y install git-all
```

Comment:
Could you try and see if doing this instead still works and gives us a faster install time with a smaller footprint?
`sudo DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install git-all`
or maybe even smaller like this...
`sudo DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install git-gui`
if that breaks something (as sometimes the "recommended" are actually "required") you can manually add them like this...
`sudo DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install git-gui thing1 thing2`
or try this...
`sudo DEBIAN_FRONTEND=noninteractive apt-get -y install git-gui`
any of those, if they work, should be noticeably smaller that what your app install does right now.  It is pulling in so much.